riotfile.py

```diff
@@ -134,7 +134,15 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="tracer",
             command="pytest {cmdargs} tests/tracer/",
-            venvs=[Venv(pys=select_pys(), pkgs={"msgpack": latest})],
+            venvs=[
+                Venv(
+                    pys=select_pys(),
+                    pkgs={
+                        "msgpack": latest,
+                        "attrs": [">=19.2.0,<20", latest],
```

Comment:
```suggestion
                        "attrs": ["==19.2.0", latest],
```

Since this is the minimum possible version we have to make sure we test with it exactly